packages/discord-bot/package.json

```diff
@@ -37,6 +37,6 @@
     "@types/node-fetch": "^2.5.7",
     "@types/pg": "^7.14.6",
     "ts-node-dev": "^1.0.0",
-    "typescript": "^4.0.5"
+    "typescript": "~3.7.2"
```

Comment:
Why the downgrade?

Reply:
It caused issues with VSC. React is locked at this version. So I made all the versions the same.

CRA 4 is out and I attempted to upgrade to that once but it was a lot of work and broke a lot of stuff so I had to roll back. I actually forgot to check in before upgrading and lost what I was working on that day.